connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/ProtobufSerializer.scala

```diff
@@ -215,10 +217,12 @@ private[sql] class ProtobufSerializer(
           duration.build()
 
       case _ =>
-        throw new IncompatibleSchemaException(
-          errorPrefix +
-            s"schema is incompatible (sqlType = ${catalystType.sql}, " +
-            s"protoType = ${fieldDescriptor.getJavaType})")
+        throw QueryCompilationErrors.cannotConvertCatalystTypeToProtobufTypeError(
+          toFieldStr(catalystPath),
```

Comment:
Please, don't quote it twice if you do that inside of `cannotConvertCatalystTypeToProtobufTypeError()` by `toSQLId`

Reply:
okay, will call toSQLType inside the error class, avoid using toFieldStr for sqlType